src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerPlaceholders.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.VisualStudio.Debugger.Metadata
+{
+    // Starting in Visual Studio 2015, Update 1, a new api exists to detect if a type is a function pointer.
```

Comment:
My understanding is that the combination of Update 1 EE with RTM debugger bits is used only for internal dogfooders of Roslyn.  They install VS 2015 RTM and run a script that updates just Roslyn binaries.  I don’t want to modify the script to also debugger Microsoft.VisualStudio.Debugger.Metadata.dll because as soon as you update one debugger binary, other dependencies, unrelated to this change force the update of others, which, in turn forces the update of others.  It’s easier to just keep the Update 1 EE working with the RTM debugger for now, as this preserves the ability of people to dogfood without destabilizing their dev machines by bringing in a bunch of other unrelated changes.

From: Manish Jayaswal [mailto:notifications@github.com]
Sent: Wednesday, September 23, 2015 4:25 PM
To: dotnet/roslyn <roslyn@noreply.github.com>
Cc: Eric Feiveson <ericfe@microsoft.com>
Subject: Re: [roslyn] Fix Roslyn EE to handle function pointers (#5412)


In src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerPlaceholders.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5412%23discussion_r40270751&data=01%7c01%7cericfe%40microsoft.com%7cf7df9d66bf7042c619fe08d2c46e37b9%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=7S2%2bnrGM2uL47x5QkHxvf7PR2PGyq03GTKPPxIOjnwk%3d>:

> @@ -0,0 +1,19 @@

> +﻿namespace Microsoft.VisualStudio.Debugger.Metadata

> +{

> +    // Starting in Visual Studio 2015, Update 1, a new api exists to detect if a type is a function pointer.

@ericfe-ms<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fericfe-ms&data=01%7c01%7cericfe%40microsoft.com%7cf7df9d66bf7042c619fe08d2c46e37b9%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=bDcw3YhGZIOa7SZv2mCKBIVE30GnSOVe4%2bF8FGKvdA4%3d> what is the setup where you are trying to use update1 EE with RTM debugger bits?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5412%2ffiles%23r40270751&data=01%7c01%7cericfe%40microsoft.com%7cf7df9d66bf7042c619fe08d2c46e37b9%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=JaHXIlTpEZhK7Tj6J502HHvS2FVHoidk%2fmkGG9LxLvg%3d>.